packages/oui-dual-list/dual-list.less

```diff
@@ -0,0 +1,209 @@
+@import './_normalize';
+@import './_mixins';
+
+// dual list
+.oui-dual-list {
+  background: @oui-color-dual-list-background;
+  border: 1px solid @oui-color-dual-list-header-background;
+  border-radius: rem-calc(3px);
+  display: flex;
+  flex-wrap: nowrap;
+  flex-direction: column;
+  height: 100%;
+  min-height: 0;
+
+  @media screen and (min-width: @oui-responsive-breakpoint-phone) {
+    flex-wrap: nowrap;
+    flex-direction: row;
+  }
+
+  &__source {
+    border-right: 1px solid @oui-color-dual-list-header-background;
+  }
+
+  &__source, &__target {
+    display: flex;
+    flex-direction: column;
+    min-height: 4rem;
+    @media screen and (min-width: @oui-responsive-breakpoint-phone) {
+      flex: 1 1 100%;
+    }
+  }
+
+  &__header {
+    display: flex;
+    flex-direction: row;
+    background: @oui-color-dual-list-header-background;
+    padding: rem-calc(10px);
+    min-height: rem-calc(40px);
+  }
+
+  &__toggle-buttons {
+    margin-left: auto;
+    border: none;
+    background-color: inherit;
+    .oui-icon {
+      font-size: @oui-dual-list-icon-size;
+    }
+    @media screen and (min-width: @oui-responsive-breakpoint-phone) {
+      display: none;
+    }
+  }
+
+  &__content {
+    display: flex;
+    flex-direction: column;
+    flex: 1;
+    padding: 0;
+    border-radius: 0 0 rem-calc(3px) rem-calc(3px);
+    min-height: 0;
+  }
+
+  &__content-body {
+    display: flex;
+    flex-direction: column;
+    min-height: 0;
+
+    & > .oui-dual-list__filter > form {
+      margin: 0;
+    }
+  }
+
+  &__content-empty {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex: 1;
+
+    p {
+      color: @oui-color-dual-list-empty-label;
+    }
+  }
+
+  &__target > &__content {
+    @media screen and (min-width: @oui-responsive-breakpoint-phone) {
+      display: flex;
+    }
+  }
+
+  &__filter {
+    position: relative;
+    padding: rem-calc(10px) @oui-dual-list-left-padding;
+
+    & > input {
+      background: @oui-color-dual-list-background;
+      border: 1px solid @oui-color-dual-list-header-background;
+      box-shadow: inset rem-calc(2px) rem-calc(2px) 1px #e9ebe7;
+      border-radius: rem-calc(3px);
+      width: 100%;
+      padding-right: 3rem;
+      height: rem-calc(40px);
+      padding-left: rem-calc(10px);
+    }
+
+    & > .oui-icon {
+      position: absolute;
+      right: @oui-dual-list-left-padding + rem-calc(10px);
+      top: 50%;
+      transform: translateY(-50%);
+      font-size: 2rem;
+    }
+  }
+
+  &__add-all-link, &__remove-all-link {
+    padding-left: @oui-dual-list-left-padding;
+    text-decoration: underline;
+    color: @oui-color-dual-list-link;
+    &:hover {
+      cursor: pointer;
+    }
+    &:hover, &:active {
+      color: @oui-color-dual-list-link;
+    }
+
+    & button {
```

Comment:
Can be replaced by `.oui-button { }`.
No need of `&` :)

Reply:
I cant. I need to remove padding and margin assigned to button inside oui-button. Adding this to oui-button does not has any effect.